app/languages/de_DE.json

```diff
@@ -1,127 +1,127 @@
 {
     "string.email": "Email",
-    "string.password": "Password",
+    "string.password": "Passwort",
     "string.login": "Login",
-    "string.reset_password": "Reset Password",
```

Comment:
"string.reset_password": "Passwort löschen",

Reply:
Please use https://crowdin.com/project/pufferpanel to submit changes